Firebase/Messaging/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# 2019-11-19 -- v4.1.9
+- [changed] Move reliable message queue delete operation in a sync queue to avoid race conditions in unit tests. (#4236)
```

Comment:
nit: replace `operation in a sync queue` -> `operations to a serial queue`

Reply:
Done.